docs/sources/datasources/_index.md

```diff
@@ -48,6 +48,18 @@ For example, this video demonstrates the visual Prometheus query builder:
 
 For general information about querying in Grafana, and common options and user interface elements across all query editors, refer to [Query and transform data]({{< relref "../panels-visualizations/query-transform-data/" >}}).
 
+## Special data sources
+
+Grafana also includes three special data sources:
+
+- **Grafana:** A built-in data source that generates random walk data and can poll the [Testdata]({{< relref "./testdata/" >}}) data source. Additionally it can list files and get other data from a Grafana installation. This can be helpful for testing visualizations and running experiments.
```

Comment:
```suggestion
- **Grafana:** A built-in data source that generates random walk data and can poll the [Testdata]({{< relref "./testdata/" >}}) data source. Additionally, it can list files and get other data from a Grafana installation. This can be helpful for testing visualizations and running experiments.
```